com.github.qarmin.szyszka.yaml

```
@@ -0,0 +1,32 @@
+app-id: com.github.qarmin.szyszka
+runtime: org.gnome.Platform
+runtime-version: '40'
+sdk: org.gnome.Sdk
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.rust-stable
+command: szyszka
+finish-args:
+- "--share=ipc"
+- "--socket=fallback-x11"
+- "--socket=wayland"
+- "--filesystem=host"
+- "--device=dri"
+build-options:
+  append-path: "/usr/lib/sdk/rust-stable/bin"
+  env:
+    CARGO_HOME: "/run/build/szyszka_gui/cargo"
```

```suggestion
    CARGO_HOME: "/run/build/szyszka/cargo"
```

This likely explain the failure.